source/dem/dem.cc

```diff
@@ -439,6 +441,15 @@ DEMSolver<dim>::particle_wall_contact_force()
   pw_contact_force_object->calculate_pw_contact_force(
     pw_pairs_in_contact, simulation_control->get_time_step(), momentum, force);
 
+  // Forces and Torques applied in each boundary
+  if (parameters.forces_torques.calculate_force_torque &&
+      (!(simulation_control->get_step_number() %
+         parameters.forces_torques.calculation_frequency) ||
+       simulation_control->get_step_number() == 1))
+    {
+      pw_contact_force_object->calculate_pw_force_torque(
+        pw_pairs_in_contact, simulation_control->get_time_step(), parameters,simulation_control->get_current_time());
```

Comment:
Why does this need the current time?

Reply:
To write this information in the text file